examples/parse_time.py

```diff
@@ -0,0 +1,94 @@
+"""
+================================================
+Parsing times with sunpy.time.parse_time
+================================================
+
+Example to show some example usage of parse_time
+"""
+##############################################################################
+# Import the required modules
+from datetime import datetime, date
+import time
+
+import numpy as np
+import pandas
+
+# Import parse_time
+from sunpy.time import parse_time
+
+
+##############################################################################
+# Suppose you want to parse some strings, ``parse_time`` can do that.
```

Comment:
Does intersphinx work here?

Reply:
@Cadair said do `"``"`

Reply:
you will need `~sunpy.time.parse_time` but it should work.